### ISSUE TITLE
Some bowls containing objects cant be put in big container

### ISSUE THREAD

**Issue by Dodg-e** [jasonrohrer/OneLifeData7#31]
Some bowls with objects dont go in cart/horsecart/boxes/chests for example pop corn, bowl of stew, bowl with corn kernels etc

**Comment by laccy** [comment]
that's not a problem, actually they all should, especially leavened dough, also kinda weird that if you pt dough in cart, gonna turn sideways in a minute

**Comment by jasonrohrer** [comment]
Yeah, no bowl is supposed to go into a cart or basket when non-empty.
